aten/src/ATen/native/cuda/Gesv.cu

```diff
@@ -46,17 +46,17 @@ void magmaGesvBatched<double>(
       dB_array, lddb, dinfo_array, batch_count, queue);
 }
 
-static magma_queue_t createMagmaQueue(const Tensor& tensor) {
-  auto& context = tensor.type().get_context();
-  magma_queue_t magma_queue;
-  magma_queue_create_from_cuda(
-      tensor.get_device(),
-      context.getCurrentCUDAStream(),
-      THCState_getCurrentBlasHandle(context.getTHCState()),
-      THCState_getCurrentSparseHandle(context.getTHCState()),
-      &magma_queue);
-  return magma_queue;
-}
+//static magma_queue_t createMagmaQueue(const Tensor& tensor) {
```

Comment:
@jithunnair-amd same as the rest of the PR. No comment has been made regarding to the rationale disabling the logic crucial to the operator here. Instead of disabling the implementation of a particular operator I'd like to have the operators themselves be disabled.